### PR TITLE
721 - Updating development docs

### PR DESCRIPTION
**Pull request by EdwardCoyle**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Making some updates to (and cleaning up) the project documentation.

**Related github/jira issue (required)**:
#721

**Steps necessary to review your pull request (required)**:
Pull this branch and read the docs

### CODE REVIEW

File: docs/QA.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# QA Documentation
+
+**TODO:** This documentation is VERY out of date and needs fixing.
```
Comment (EdwardCoyle) [inline docs/QA.md]: Wouldn't mind some feedback here on the links... I know the ones here are out of date but not sure if we should get rid of them, or replace them with something more relevant.

**Comment by EdwardCoyle** [comment]
@tmcconechy yep, just added a link for it.

File: docs/QA.md (hunk 1 above)
Comment (tmcconechy) [reply docs/QA.md]: Maybe @CindyMercadoReyes can add some notes for us?

Comment (clepore) [reply docs/QA.md]: 👍 I will make a note of this and we can make it a separate initiative.

**Comment by clepore** [comment]
Is there any steps we could write about between install it from npm and then running your project? Like physically do we do a `<script src="node_modules...">` and then add a `<link rel>` tag too? I know it might be a bit nitty gritty, but those steps (maybe in a docs/QUICKSTART.md file) could help with the basics...?

**Comment by tmcconechy** [comment]
Also run `npm run mdlint`. The tests that are failing should be fixed in my PR in a few minutes...

File: docs/QA.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Browser Testing - IE11, Edge, Firefox, Safari, Chrome, IOS  (Phone and Tablet), Android (Phone and Tablet), Windows Touch (Surface)
+- Touch Device Support Testing
+- Accessibility Testing according to [WCAG Accessibility Standards](http://www.w3.org/TR/WCAG20/)
+- Developer Testing - Can the Components be easily coded from the documentation, developer Web Driver.IO tests ect.
+- Testing against requirements
+
+## Related Testing Tools
+
+- Tester can grab latest code from github (Generally One Month Cycles for versions/QA).
+- tester would go through every Html page example opening it in:  IE11, Edge Firefox, Safari, Chrome, IOS  (Phone and Tablet), Android (Phone and Tablet), Windows Touch (Surface)
```
Comment (davidcarlsonberg) [inline docs/QA.md]: Capitalize this bullet

File: docs/QA.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- Cross check designs with <https://docs.google.com/spreadsheets/d/1Z9mmUZJlAqkCLy8vyie3AjRKUR4SVKuHGyFiQiGm98A/edit?pli=1#gid=989468169>
+- Tests for Accessibility
+    1. Run Wave Toolbar <https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related> and <https://addons.mozilla.org/en-US/firefox/addon/wave-toolbar/>
+    1. Run Squizz – Html Sniffer <http://squizlabs.github.io/HTML_CodeSniffer/>
+    1. Test for Keyboard against <http://access.aol.com/dhtml-style-guide-working-group/>
+    1. Test with screen readers (voice over, nvda and jaws), If only one Jaws is most critical
```
Comment (davidcarlsonberg) [inline docs/QA.md]: Period after parentheses.
Also: “If only one, Jaws is the most critical.”

**Pull request merged**